src/test/utils/EigenLayerUnitTestBase.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "src/contracts/permissions/PauserRegistry.sol";
+import "src/test/mocks/Reenterer.sol";
+import "forge-std/Test.sol";
+
+abstract contract EigenLayerUnitTestBase is Test {
+    Vm cheats = Vm(HEVM_ADDRESS);
+
+    PauserRegistry public pauserRegistry;
+    ProxyAdmin public eigenLayerProxyAdmin;
+    Reenterer public reenterer;
+
+    mapping(address => bool) public addressIsExcludedFromFuzzedInputs;
+
+    address public constant pauser = address(555);
+    address public constant unpauser = address(556);
+
+    // Helper Functions/Modifiers
+    modifier filterFuzzedAddressInputs(address fuzzedAddress) {
+        cheats.assume(!addressIsExcludedFromFuzzedInputs[fuzzedAddress]);
+        _;
+    }
+
+    function setUp() public virtual {
+        address[] memory pausers = new address[](1);
+        pausers[0] = pauser;
+        pauserRegistry = new PauserRegistry(pausers, unpauser);
+        eigenLayerProxyAdmin = new ProxyAdmin();
+        reenterer = new Reenterer();
```

Comment:
Don't think reenter needs to be declared here since we use it in a minority of unit tests - only SM, DM, and EPM have a reentrancy guard. IMO this contract should only be declared in the sub-test contract that is testing reentrancy

Reply:
Removed and fixed